lib/kvdb/ikvdb.c

```diff
@@ -1855,46 +1954,6 @@ ikvdb_kvs_close(struct hse_kvs *handle)
     return err;
 }
 
-merr_t
-ikvdb_storage_info_get(
-    struct ikvdb *                handle,
-    struct hse_kvdb_storage_info *info)
-{
-    struct ikvdb_impl *self = ikvdb_h2r(handle);
-    struct mpool *     mp;
-    struct mpool_stats stats = {};
-    merr_t             err;
-    uint64_t           allocated, used;
-
-    mp = ikvdb_mpool_get(handle);
-    err = mpool_stats_get(mp, &stats);
-    if (ev(err))
-        return err;
-
-    info->total_bytes = stats.mps_total;
-    info->available_bytes = stats.mps_available;
-
-    info->allocated_bytes = stats.mps_allocated;
-    info->used_bytes = stats.mps_used;
-
-    /* Get allocated and used space for kvdb metadata. Allocated and used are
-     * the same.
-     */
-    err = kvdb_meta_usage(self->ikdb_home, &used);
-    if (ev(err))
-        return err;
-    info->allocated_bytes += used;
-    info->used_bytes += used;
-
-    err = cndb_usage(self->ikdb_cndb, &allocated, &used);
```

Comment:
Think we need to include cndb and wal usage in mpool_mclass_info_get(). Please do not worry about it for now, I will take care of it in my upcoming PR.